<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntiAliasingPage</class>
 <widget class="QWidget" name="AntiAliasingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelNoAntiAnliasing">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioNoAntiAnliasing">
       <property name="text">
        <string>No AntiAliasing</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelAntiAnliasing">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAntiAnliasing">
       <property name="text">
        <string>AntiAliasing</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
